src/Bundle/Listener/CacheActionListener.php

```diff
@@ -119,7 +117,7 @@ public function onKernelResponse(FilterResponseEvent $event)
 
             $cacheKey = $request->attributes->get('cache_key');
             $ttl      = $response->headers->getCacheControlDirective('max-age');
-            $this->cacheService->set($cacheKey, $response->getContent(), $ttl);
+            $this->cacheService->setConcurrent($cacheKey, $response->getContent(), $ttl);
```

Comment:
The only work i do is to take code from our *working* internal version of this bundle to put it here.
Do we really change this behaviour, with the risk that the usage change ?